README.md

```diff
@@ -1,56 +1,181 @@
+
+- [Not available at the moment]()
+
+## 💻 Getting Started
+
+> Describe how a new developer could make use of your project.
 
 To get a local copy up and running, follow these steps.
+Clone this repository to your desired folder:
+
+````sh
+  cd Hello-Project
+  git https://github.com/ghermaico135/Hello-Project
+
+### Prerequisites
+
+In order to run this project you need:
+
+install a vscode text editor or any other editor
+chrome
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this commands:
+
+```sh
+  cd Hello-Project
+  git https://github.com/ghermaico135/Hello-Project
+````
+
+### Install
+
+Install this project with:
+
+Run this command:
 
-Prerequisites
-In order to run this project you need: to have latest chrome browser installed
+```sh
+  cd hello-microversehttps://github.com/ghermaico135/Hello-Project
+  node.js v12 and above
+```
 
-Setup
-Clone this repository to your desired folder:"github/ghermaico135"
-And try to open in any editor however recommended is vscode
+### Usage
 
-Usage
-To run the project, execute right click open in chrome browser
+To run the project, execute the following command:
 
-#Run tests
-I used html-css linter to run the test for minimize errors and to improve performance
-#Author
-Author miki
-GitHub: ghermaico135
-Twitter: @mikidarling
-LinkedIn: miki
-🔭 Future Features
-new feature-1 will be added to make more dynamic by linking with javascript
+Example command:
+
+```sh
+  use git bash to open in Vscode
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+Example command:
+
+```sh
+  Run "npx stylelint "**/*.{css,scss}" --fix " to fix linters
+```
+
+### Deployment
+
+You can deploy this project using:
+
+Github Page
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/ghermaico135)
+- Twitter: [@twitterhandle](https://twitter.com/@mikidarling)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/miki)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[add linters for html, css]**
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
 
-#Contributing
 Contributions, issues, and feature requests are welcome!
 
-🙏 Acknowledgments
-Give credit to everyone in the microverse community .
+Feel free to check the [issues page](../../issues/).
+
+## ⭐️ Show your support
+
+> Write a message to encourage readers to support your project
+
+If you like this project, please give your rating or suggest to me how to make it better
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank my stand-up team and reviewer
 
-    I would like to thank...
+## 📝 License <a name="MIT license"></a>
 
-📝 License
-<a href="./MIT licencse"> MIT licencse</a>
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] I have noticed that your MIT is broken. The MIT license gives users :person_curly_hair:  express permission to reuse code for any purpose. Kindly ensure to fix this ☺️.